Packs/AzureDevOps/Integrations/AzureDevOps/AzureDevOps.yml

```diff
@@ -224,12 +224,12 @@ script:
     name: azure-devops-user-remove
     outputs: []
   - arguments:
-    - description: The name or ID of the project.
+    - description: The name or ID of the project. Default value will be config param, user can supply a different value.
       name: project
-      required: true
-    - description: The repository ID of the pull request's target branch. A repository ID can be obtained by running the 'azure-devops-repository-list' command.
+      required: false
+    - description: The repository ID. A repository ID can be obtained by running the 'azure-devops-repository-list' command. Default value will be config param, user can supply a different value.
       name: repository_id
-      required: true
+      required: false
```

Comment:
is this intended? same for all `required` changes

Reply:
Yes, for the PB's